tests/commands/test_edit.py

```diff
@@ -9,7 +10,7 @@
 
 
 @pytest.mark.library_setup(settings={
-    "editor": "python {} sed".format(script)
+    "editor": "{} {} sed".format(sys.executable, script)
```

Comment:
This fix is definitely useful! If you want to put it in another PR, we can get it in right away.

Reply:
Will do. After this, I am preparing a PR that allows for usage in vanila windows installations (no unix commands there). They will come in handy when releasing!